strapi/strapi
# fix(content-releases): align columns in the Release Details page

### What does it do?

It sets the width of some columns in the Release Details table to align the different tables that contains or not locales

### Why is it needed?

Otherwise, we have different columns with different widths based on their content.

### How to test it?

- Add some entries to a release from a collection without internationalization
- Add other entries to the same release from a collection with internationalization
- Open the Release Details page of this release and check the alignment of the columns

### Related issue(s)/PR(s)

CS-518
